test/data/schemas.test.ts

```diff
@@ -123,6 +126,10 @@ function isIncentiveAmountValid<T extends StateIncentive>(
       return amount.number >= 0 && amount.number <= 1 && !amount.unit;
     case 'dollars_per_unit':
       return amount.unit !== undefined;
+    case 'other':
+      return true;
+    case 'unknown':
+      return true;
```

Comment:
there were a few weird cases that led to adding these.

1. NV-17 is a program for a [free clothes dryer](https://www.nvenergy.com/save-with-powershift/qualified-appliance-replacement). I marked this as "other". as far as I can tell on the website there isn't a monetary value, everywhere just says "free ENERGY STAR® electric clothes dryer"
2. NV-26 - program for a [free smart thermostat](https://www.nvenergy.com/save-with-powershift/smart-thermostat). I also marked this as "other"
3. NV-57 and NV-58 had no information about how much the rebate was actually for ([example application](https://www.rrelectric.com/wp-content/uploads/Residential-Insulation-Measures-Application.pdf)). it pretty much just says contact office to get more details... I marked these as "unknown".

let me know if this is the wrong approach

Reply:
For free items, we express those as having amount type percent and the value is 1, representing 100% of costs (100% of eligible costs is what you can put in the Rebate Value column).

For NV-57 and NV-58, for similar cases in the past we've made the decision not to show those incentives at all in the JSON (see https://docs.google.com/document/d/18kkfkuz1Wn6_FYWid0U1v5VsJ8FUTELzzqhWJk1sbfM/edit, question #5).

I have seen that "unknown" is a handy value to have, at least for the spreadsheets, so I might add that later, but for now I don't think we want to support it and I don't think it should be valid for API data.

Reply:
Didn't see Owen's comment before adding mine, but looks like we're generally aligned.